### PR TITLE
Remove outdated certificate and update certificates for puppeteer

### PR DESCRIPTION
**Pull request by Surfict**
## What do these changes do?

Remove outdated certificate and update certificates for puppeteer

### CODE REVIEW

**Review by mguidon — APPROVED** [review]
Looks good to me

**Review by GitHK — APPROVED** [review]
ok for me as well

File: tests/e2e/docker/Dockerfile [hunk 1]
```diff
@@ -71,6 +71,8 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /
 RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
   chmod +x /usr/local/bin/docker-compose
 
+RUN grep -q mozilla/DST_Root_CA_X3.crt  "/etc/ca-certificates.conf" && sed -e '/mozilla\/DST_Root_CA_X3.crt/s/^/!/g' -i /etc/ca-certificates.conf && update-ca-certificates
```
Comment (sanderegg) [inline tests/e2e/docker/Dockerfile:74]: please use long option names in Dockerfiles (i.e. instead of ```-i``` use ```--in-place```

**Review by sanderegg — APPROVED** [review]
please use long option names. Great!

File: tests/e2e/docker/Dockerfile (hunk 1 above)
Comment (pcrespov) [reply tests/e2e/docker/Dockerfile:74]: MINOR: use multiline, i.e. with ``\``, to visualize easier changes and options

**Comment by pcrespov** [comment]
@Surfict any update on this?

File: tests/e2e/docker/Dockerfile (hunk 1 above)
Comment (pcrespov) [reply tests/e2e/docker/Dockerfile:74]: @Surfict ??

**Pull request merged**